omega/omega-connector/omega-connector-grpc/src/main/java/org/apache/servicecomb/saga/omega/connector/grpc/tcc/GrpcCoordinateStreamObserver.java

```diff
@@ -6,33 +6,34 @@
  * (the "License"); you may not use this file except in compliance with
```

Comment:
Hi, I think You need to change the template.

Reply:
sorry, I haven't understand what it mean.